src/nurse/tests/management/commands/test_notifications.py

```diff
@@ -4,15 +4,33 @@
 from django.test import override_settings
 
 from nurse.management.commands import _notifications
+from nurse.models import Prescription, UserOneSignalProfile
 
 ONESIGNAL_APP_ID = "test_app_id"
 ONESIGNAL_API_KEY = "test_api_key"
 client_path = "nurse.management.commands._notifications.Client"
 
 
 class TestNotifications:
+    @pytest.mark.django_db
     def test_notify(self):
-        with mock.patch(
+        # Mock data
+        mock_prescriptions = [mock.Mock(), mock.Mock()]
+        mock_users = {
+            mock_prescriptions[0]: mock.Mock(),
+            mock_prescriptions[1]: mock.Mock(),
+        }
+        mock_subscription_ids = ["123", "456"]
+
+        with mock.patch.object(
+            Prescription.objects, "expiring_soon", return_value=mock_prescriptions
+        ), mock.patch(
+            "nurse.utilities.get_users_for_prescriptions", return_value=mock_users
+        ), mock.patch.object(
```

Comment:
I'd rather hit the database for real to really test that flow top-down

Reply:
done